### ISSUE TITLE
Certificate’s generated upon installation by default not being trusted by the organization

### ISSUE THREAD

**Issue by WorkBlake** [MicrosoftDocs/OfficeDocs-Exchange#293]
Other organizations might have different methods to how they control certificates and utilize the trusted root authority. For us there is essentially no way to have the Certificates generated by default be trusted unless they are pushed to all machines by Group Policy (as determined by MS support on a critsit case). How do others get around this? Even if we had our Desktop Engineering team ready to go as soon as our installation was done, there would have still been nearly just as many people getting this new certificate prompt in Outlook in our environment.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e9e0968e-9590-59a1-83c8-07c218bcf8fd
* Version Independent ID: 42d25a51-bb1a-3d51-75fe-85731236aa04
* Content: [Exchange Server prerequisites](https://docs.microsoft.com/en-us/Exchange/plan-and-deploy/prerequisites?view=exchserver-2016)
* Content Source: [Exchange/ExchangeServer/plan-and-deploy/prerequisites.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeServer/plan-and-deploy/prerequisites.md)
* Product: **exchange-server-it-pro**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by AndreaBarr** [comment]
@WorkBlake Thank you for this feedback

I have gotten this over to the Exchange writing team for investigation. 

Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.

**Comment by murrato1** [comment]
It's basically a timing issue.  Here's one method I've used to get around the problem:
https://gallery.technet.microsoft.com/scriptcenter/Set-AutoDiscoverServiceInte-3930e163?redir=0

**Comment by chrisda** [comment]
@WorkBlake, upon inspection, there's really no place to talk about certificates in this specific topic.

Based on your Issue #289 , I added a link near the top of the topic:

To actually install Exchange, see [Deploy new installations of Exchange](https://docs.microsoft.com/Exchange/plan-and-deploy/deploy-new-installations/deploy-new-installations).

I've made sure that topic and the child "delegate installation", "install mailbox role" and "unattended installs" topics all contain a link to "Exchange post-installation tasks", which talks in depth about configuring a certificate.
